zubhub_frontend/zubhub/src/assets/js/styles/components/autocomplete/optionStyles.js

```diff
@@ -1,50 +1,83 @@
 const styles = theme => ({
   option: {
     display: 'flex',
-    flexFlow: 'row nowrap',
+    flexFlow: 'row',
     alignItems: 'center',
-    justifyContent: 'space-between',
+    justifyContent: 'flex-start',
+    gap: '8px',
     width: '100%',
     height: '100%',
-    margin: '-10px -15px',
-    padding: '10px 15px',
     textDecoration: 'none',
     color: 'black',
   },
   infoWrapper: {
     display: 'flex',
-    flexFlow: 'row nowrap',
     alignItems: 'center',
-    maxWidth: '50%',
+    flexFlow: 'row',
+    flexGrow: 1,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    //first approach 
```

Comment:
the first approach was keeping the username at right for big screens. You can see the comparison [here](https://github.com/unstructuredstudio/zubhub/issues/419#issuecomment-1772265140)

Reply:
Lets mover username below. I also wanted to highlight that for smaller screens where the title gets split in two lines and we get one more username line below, we need to align all this within the height of the image so everything looks proportional. At this time, it is bleeding from top and bottom a bit. Take design help if needed.